src/complex/Common/TypesUtility.hpp

```diff
@@ -57,4 +57,49 @@ constexpr NumericType GetNumericType() noexcept
     static_assert(AlwaysFalse_v<T>, "complex::GetNumericType: Unsupported type");
   }
 }
+
+/**
+ * @brief Returns a string representation of the passed in complex::NumericType
+ * @param numericType
+ * @return String or 'UNKNOWN TYPE' if the type is not known
+ */
+std::string NumericTypeToString(complex::NumericType numericType)
+{
+  switch(numericType)
+  {
+  case NumericType::int8: {
+    return "Int8";
+  }
+  case NumericType::uint8: {
+    return "UInt8";
+  }
+  case NumericType::int16: {
+    return "Int16";
+  }
+  case NumericType::uint16: {
+    return "UInt16";
+  }
+  case NumericType::int32: {
+    return "Int32";
+  }
+  case NumericType::uint32: {
+    return "UInt32";
+  }
+  case NumericType::int64: {
+    return "Int64";
+  }
+  case NumericType::uint64: {
+    return "UInt64";
+  }
+  case NumericType::float32: {
+    return "Float32";
+  }
+  case NumericType::float64: {
+    return "Float64";
+  }
+  default:
+    return fmt::format("UNKNOWN TYPE:{}", numericType);
+  }
+}
+
```

Comment:
```suggestion
/**
 * @brief Returns a string representation of NumericType
 * @param numericType
 * @return
 */
inline constexpr const char* NumericTypeToString(NumericType numericType)
{
  switch(numericType)
  {
  case NumericType::int8: {
    return "int8";
  }
  case NumericType::uint8: {
    return "uint8";
  }
  case NumericType::int16: {
    return "int16";
  }
  case NumericType::uint16: {
    return "uint16";
  }
  case NumericType::int32: {
    return "int32";
  }
  case NumericType::uint32: {
    return "uint32";
  }
  case NumericType::int64: {
    return "int64";
  }
  case NumericType::uint64: {
    return "uint64";
  }
  case NumericType::float32: {
    return "float32";
  }
  case NumericType::float64: {
    return "float64";
  }
  default:
    throw std::runtime_error("complex::NumericTypeToString: Unknown NumericType");
  }
}

```
The function should be `inline`. And we can make it `constexpr` as well.